source/fundamentals/connection/connect.txt

```diff
@@ -87,11 +87,25 @@ Other Connection Targets
 Connect to Atlas
 ~~~~~~~~~~~~~~~~
 
-In the following example, the driver uses a sample connection URI to
-connect to a MongoDB instance on Atlas, using placeholders for your
-username and password. In this example, the client uses
-the :ref:`Stable API <csharp-stable-api>` feature, though
-you can connect to Atlas without this feature.
+To connect to an Atlas deployment, you need to create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the ``MongoClient``
+constructor. To specify your connection URI, pass it to the ``FromConnectionString()``
+method, creating a ``MongoClientSettings`` instance. To specify any other
+settings, set the relevant fields of the settings object.
+
+To learn more about connection options, see the
+:ref:`Connection Options page <csharp-connection-options>`.
+
+You can set the {+stable-api+} version as a setting to avoid
+breaking changes when you upgrade to a new server version. To 
+learn more about the {+stable-api+} feature, see the :ref:`{+stable-api+} page
+<csharp-stable-api>`.
+
+The following code shows how you can instantiate a client that uses the
+{+stable-api+} version and a sample connection string with placeholders for
+your username and password. Instantiating the client creates a
+connection to MongoDB:
```

Comment:
Issue:

1) I think the style guide mentions that when using a colon, it should introduce the content that follows it (style/punctuation). I think "Instantiating the client creates a connection to MongoDB:" is describing what happens when you instantiate a client rather than what the code example shows. I think the first sentence describes the code sample, but the second one separates it and therefore the colon is not suitable.

2) I think the conjunction is ambiguous in this sentence. One version reads as  "...code shows how you can... instantiate a client... and a sample connection string" and another as "... instantiate a client that uses the Stable API... and [uses] a sample connection string." I think the sentence structure can be changed to avoid the ambiguity.

Suggestion:

Perhaps simplifying the message could resolve both issues.

E.g.

```suggestion
The following code shows how you can specify the connection string and the Stable API client option when connecting to a MongoDB deployment:
```

Reply:
The code also includes a confirmation of successful connection now, so I will integrate this suggestion with that new description as well.